src/main/kotlin/no/nav/klage/dokument/gateway/DefaultKabalSmartEditorApiGateway.kt

```diff
@@ -62,10 +62,6 @@ class DefaultKabalSmartEditorApiGateway(
         kabalSmartEditorApiClient.deleteDocument(smartEditorId)
     }
 
-    fun deleteDocumentAsSystemUser(smartEditorId: UUID) {
```

Comment:
Jeg slettet ikke denne i går fordi jeg tror vi trenger den når vi skal bruke cron-jobb for å slette smartdokumenter.

Reply:
Godt poeng, skal gjøre om den.